test/extended/cpu_partitioning/crio.go

```diff
@@ -313,6 +313,9 @@ func collectContainerInfo(ctx context.Context, oc *exutil.CLI, node corev1.Node)
 			if outStdErr != "" {
 				return fmt.Errorf("err execing command %s", outStdErr)
 			}
+			if isClusterCPUPartitioned && len(out) == 0 {
+				return fmt.Errorf("err execing command, script result was empty but expected values")
+			}
```

Comment:
we observed this happening in the wild? what was the context?

Reply:
Yeah, that was my bad, updated the description with info on that info. In this example job you'll see it flake with a an empty array

https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-ovn-cpu-partitioning/1685720543094902784